groups/committee-steering/groups.yaml

```diff
@@ -43,12 +46,12 @@ groups:
     owners:
       - augustus@cisco.com
       - bentheelder@google.com
-      - cblecker@gmail.com
-      - ctadeu@gmail.com
       - k8s@auggie.dev
       - killen.bob@gmail.com
+      - maszulik@redhat.com
       - pal.nabarun95@gmail.com
-      - tpepper@vmware.com
+      - patrick.ohly@intel.com
+      - roollingstone@gmail.com
```

Comment:
@pacoxu -- Please confirm preferred email address for Steering comms

Reply:
paco.xu@daocloud.io
I may use this one as preferred email for the networking issue in China.

Reply:
Thanks @pacoxu! I've updated your email address.